processes/release_processes/README.MD

```diff
@@ -53,7 +53,7 @@ Make or copy the following files into a temporary `release/` directory outside t
   "publish_workflow.githubToken": "<<GITHUB TOKEN VALUE>>",
   "publish_workflow.majorRelease": true,
   "publish_workflow.publishHomebrew": false,
-  "publish_workflow.publishDocker": "broadinstitute/cromwell-publish:latest",
+  "publish_workflow.publishDocker": "broadinstitute/cromwell-publish:ubuntu-nobrew",
```

Comment:
Do we want to do the naming shuffle and formally call this the latest version of this image?

Reply:
Agreed. I wanted to make sure the release well and truly works before doing so - e.g. make sure we don't need to put the SBT path in the workflow.